### PR TITLE
Create an eventbridge terraform module

### PR DESCRIPTION
**Pull request by amit-mazor**
### Adding a new eventbridge terraform module 
* Added an example
* Updated readme and usage example
* Deleted unused var 'application_name'

### CODE REVIEW

File: modules/eventbridge/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+* singapore
+* ireland
+* india
+* stockholm
+
+### All of the regions must be written with lower-case letters. 
+
+| Region    | Metrics Endpoint
+|-----------|-----------------------------------------------------------------|
+| us        | `https://aws-events.coralogix.us/aws/event`                |
```
Comment (oded-dd) [inline modules/eventbridge/README.md:83]: This should be changed to `https://ingress.coralogix.com/aws/events
Same should be for privateLink

File: modules/eventbridge/main.tf [hunk 2]
```diff
@@ -0,0 +1,116 @@
+
+provider "aws" {
+  region = data.aws_region.current_region.name
+}
+locals {
```
Comment (oded-dd) [inline modules/eventbridge/main.tf:5]: Should move to variables.

File: modules/eventbridge/README.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
+# EventBridge_module_toCoralogix
+Configuration in this directory creates eventbridge to send the aws events to your coralogix account.
+
+## Metrics - Usage
+### Provision a eventbridge stream for streaming events to Coralogix  Delivery Stream
+
+#### Creating api destination
+```
+resource "aws_cloudwatch_event_api_destination" "api-connection" {
```
Comment (oded-dd) [inline modules/eventbridge/README.md:9]: The example should be module, and include the defaults.

File: modules/eventbridge/README.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+  description                      = "EventBridge Api destination to Coralogix"
+  invocation_endpoint              = local.endpoint_url[var.coralogix_region].url
+  http_method                      = "POST"
+  invocation_rate_limit_per_second = 300
+  connection_arn                   = aws_cloudwatch_event_connection.event-connectiong.arn
+}
+```
+#### Connecting api destination to your coralogix account
+```
+esource "aws_cloudwatch_event_connection" "event-connectiong" {
```
Comment (oded-dd) [inline modules/eventbridge/README.md:20]: The example should be module, and include the defaults.

File: modules/eventbridge/README.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+```
+
+### Delivering all Eventbridge events
+Provision a Eventbridge delivery rule with [Events from AWS services](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-service-event.html).
+and sends the events to Coralogix:
+### Creating Rule for classify the events we want to get
+You can send all event or select certain services that you interested.
+
+```
+resource "aws_cloudwatch_event_rule" "console" {
```
Comment (oded-dd) [inline modules/eventbridge/README.md:41]: The example should be module, and include the defaults.

File: modules/eventbridge/README.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+```
+
+### Delivering selected Eventbridge events
+
+```
+resource "aws_cloudwatch_event_target" "example" {
+  arn  = aws_cloudwatch_event_api_destination.api-connection.arn
+  rule = aws_cloudwatch_event_rule.console.name
+  role_arn = var.role_to_use
+}
```
Comment (oded-dd) [inline modules/eventbridge/README.md:61]: Should be part of a module

**Review by oded-dd — CHANGES_REQUESTED** [review]

File: modules/eventbridge/main.tf (hunk 2 above)
Comment (amit-mazor) [reply modules/eventbridge/main.tf:5]: its on purpose, i did it also in firehose, we have the `coralogix_region` variable, and then this local var inserts the relevant endpoint based on the region the customer put . see line 94

**Pull request merged**